geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnection.java

```diff
@@ -1102,6 +1102,10 @@ public void removeUserAuth(Message message, boolean keepAlive) {
     }
   }
 
+  protected ClientUserAuths getClientUserAuths() {
```

Comment:
probably mark this as @VisibleForTesting

Reply:
Thanks for the suggestion. Fixed.